.github/actions/sonar-analysis/action.yaml

```
@@ -0,0 +1,100 @@
+name: "Sonar Analysis"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)